.ci/.env.prd

```diff
@@ -28,7 +28,7 @@ REACT_APP_CONTROLLER_ADDRESS=0x6843291bd86857d97f0d269e698939fb10d60772
 REACT_APP_REGISTRAR_ADDRESS=0x2a187453064356c898cae034eaed119e1663acb8
 REACT_APP_MARKETPLACE_GRAPH_URL=https://api.thegraph.com/subgraphs/name/decentraland/marketplace
 REACT_APP_IPFS_URL=https://ipfs.infura.io:5001/api/v0/add?pin=false
-REACT_APP_LAND_POSITION_URL=https://play.decentraland.org?position=
+REACT_APP_EXPLORER_URL=https://play.decentraland.org
```

Comment:
Must be `.zone ` for the `WITH_COLLECTION` thing

Reply:
It's being handled here, so we don't have to have two env vars: https://github.com/decentraland/builder/pull/1429/files#diff-294880f7daea062fae2add927da5e496ac51f396bd89cd960673f9da05e3912cR23